common/src/commonMain/resources/MR/fr/strings.xml

```diff
@@ -690,4 +690,25 @@
     <string name="skills_messages_no_skills_in_compendium_subtext">Il n\'y a pas encore de compétences dans votre compendium.</string>
     <string name="authentication_messages_google_account_collision">Il existe déjà un compte WFRP Master associé à un compte Google donné.
 \nVoulez-vous vous connecter à ce compte à la place \?</string>
+    <string name="trappings_expression_constants_weapon_range">Arme de portée</string>
```

Comment:
@TheoKDK I suggest we change this to something like `ArmeDePortee` to match other constants/variables. Is that legible in French?

Reply:
No problem, I'll make the change now